x-pack/plugins/inference/public/chat_complete.ts

```diff
@@ -22,12 +39,18 @@ export function createChatCompleteApi({ http }: { http: HttpStart }): ChatComple
       functionCalling,
     };
 
-    return from(
-      http.post('/internal/inference/chat_complete', {
-        asResponse: true,
-        rawResponse: true,
+    if (stream) {
+      return from(
+        http.post('/internal/inference/chat_complete/stream', {
```

Comment:
We don't have platform-wide guidance, so consider this an optional point of discussion. Some of our "RPC" style endpoints that don't follow traditional REST/CRUD operations have their final path segment prefixed with an underscore. In this example, `/internal/inference/chat_complete/stream` would become `/internal/inference/chat_complete/_stream`.

Do you have an opinion one way or the other on a format we should take as a team? I'll ultimately defer to you, but I thought this was something originally born out of Core at some point in the distant past.

Reply:
AFAIK we never recommended or even encouraged those `_{endpoint}` path for non crud API. 

I think that `_` prefix is a remain from a very, very old era, and was more to flag "internal' APIs before we started using that `/internal` prefix.

Now tbh, for internal apis under the `/internal` prefix, I don't really see the point given it's meant to only be used by developers (or arguably only the owners - to reexpose via the browser plugin contract) so I don't think there is any value in that particular kind of path conventions.

Now, if you have a strong preference, I would be fine adding that underscore.

Reply:
> I think that _ prefix is a remain from a very, very old era, and was more to flag "internal' APIs before we started using that /internal prefix. Or maybe we just copied ES at some point. IDK.

Call me an old timer without calling me an old timer 😄 

> Now, if you have a strong preference, I would be fine adding that underscore.

I do not have a strong preference. As I said, I'm happy to defer to you, I just wanted to raise this for discussion.